src/System.Management.Automation/engine/Modules/ModuleSpecification.cs

```diff
@@ -136,7 +136,7 @@ internal static Exception ModuleSpecificationInitHelper(ModuleSpecification modu
             string message;
             if (badKeys.Length != 0)
             {
-                message = StringUtil.Format(Modules.InvalidModuleSpecificationMember, "ModuleName, ModuleVersion, RequiredVersion, GUID", badKeys);
+                message = StringUtil.Format(Modules.InvalidModuleSpecificationMember, "ModuleName, ModuleVersion, MaximumVersion, RequiredVersion, GUID", badKeys);
```

Comment:
Please add to PR Summary:
```
Also fixing following error message to include 'MaximumVersion' in the list of 'valid members':
  <data name="InvalidModuleSpecificationMember" xml:space="preserve">
    <value>The hashtable describing a module contains one or more members that are not valid.  The valid members are ({0}). Remove the members that are not valid ({1}), then try again.</value>
  </data>
```

Reply:
Done.

Reply:
Ideally this should use `$"{nameof(...)}"` for the bad members

Reply:
nameof(badKeys) returns 'badKeys', but we need its actual value.
Or am I missing something?